test/Microsoft.AspNetCore.Server.Kestrel.FunctionalTests/RequestTests.cs

```diff
@@ -27,6 +25,8 @@ namespace Microsoft.AspNetCore.Server.Kestrel.FunctionalTests
 {
     public class RequestTests
     {
+        const int semaphoreWaitTimeout = 2500;
```

Comment:
Nit: `private const int`.  I would also prefix the name with `_`, but I'm not sure if this is a project convention or just my personal convention.